src/cdomains/valueDomain.ml

```diff
@@ -324,7 +324,8 @@ struct
               a (* probably garbage, but this is deref's problem *)
               (*raise (CastError s)*)
             | SizeOfError (s,t) ->
-              M.warn_each("size of error: " ^  s ^ "\n");
+              M.warn_each (M.LogEvent.may (M.EventType.Unknown ("size of error: " ^  s)));
+              (* old warning: M.warn_each("size of error: " ^  s ^ "\n");*)
```

Comment:
Why is this commented out old version still here?

Reply:
I didn't touch the commented out code as that's not something included in the compilation so if I made a typo in there it would be just a mess if somebody wanted to use it later. I'm not even sure why these commented out are there anyway?

Should the commented out lines be also changed then? There's only a few with the warnings.

Reply:
From the diff it looks like the comment is an added 'backup copy' of the removed line.